lib/Language/Haskell/Stylish/Step/Imports'.hs

```diff
@@ -0,0 +1,198 @@
+{-# LANGUAGE BlockArguments #-}
+{-# LANGUAGE DoAndIfThenElse #-}
+{-# LANGUAGE LambdaCase #-}
+module Language.Haskell.Stylish.Step.Imports'
```

Comment:
Having a file with `'` is unfortunately bound to cause trouble in loose `grep` / `find` / ... processing of the source code, we should rename this to `Imports2.hs` or `ImportsGHC.hs`.

Reply:
Fixed!